### DISCUSSION TITLE
High refresh rate

### DISCUSSION THREAD

**Discussion by ShadowTux** [bromite/bromite#920] [Q&A]
Hello!

Is there a way to enable high refresh rate support? I haven't been able to find it. I currently own 120hz device and it would be nice to use this browser with it. If not I'll do a feature request on this.

**Comment by csagan5** [comment]
Such feature requests are not accepted; you need to verify what is the behaviour of Chromium and whether it works there or not. If it works on Chromium and not on Bromite then you can submit a bug.
